Dockerfile

```diff
@@ -28,7 +28,7 @@ ENV EFS_CLIENT_SOURCE=$client_source
 
 RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} make aws-efs-csi-driver
 
-FROM amazonlinux:2
+FROM amazonlinux:2.0.20211201.0
```

Comment:
we should not need to pin it (the latest at buildtime should get picked up )

Reply:
Thanks, let me still keep `FROM amazonlinux:2`